pkg/plugin/clientmgmt/manager.go

```diff
@@ -77,6 +79,12 @@ type Manager interface {
 	// GetDeleteItemAction returns the delete item action plugin for name.
 	GetDeleteItemAction(name string) (velero.DeleteItemAction, error)
 
+	// GetItemBlockActions returns all v1 ItemBlock action plugins.
+	GetItemBlockActions() ([]ibav1.ItemBlockAction, error)
```

Comment:
this is a standard, but would suggest changing name here to avoid confusion, since 2 functions have very similar name

for example, changing here to `GetAllItemBlockActions`

Reply:
I'd rather not make this plugin inconsistent with every other plugin. Every other plugin type uses singular name to get one, and plural name to get all.